agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4405,6 +4408,12 @@ bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMe
     }
 }
 
+
+bool slave_thread::handle_beacon_metrics_query_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    return false;
+}
+
```

Comment:
Looking at the [entire commit](https://github.com/prplfoundation/prplMesh/pull/1114/commits/0bd383714bbdc6409d1d28366612f1f14ac706ec), it seems like there are far more changes then described in the commit messages.
If you decided to reorder the `switch/case` alphabetically, at least do it in a separate commit.

Reply:
Remember that this is a draft PR, so you can't reliably give feedback on commit messages, commit splitting or commit ordering.

Reply:
I'll see how I squash, unite, remove or whatever some of the commits.
The goal - clean history.
I chose to develop without too much care about the amount of commits, changing back and forth and so on. I just worked on the code and let git save the changes for me.
I'll fix it into another branch before merging to master.